app/models/crime_application.rb

```diff
@@ -37,7 +37,13 @@ def passporting_benefit?
     means_passport.include?(Types::MeansPassportType['on_benefit_check'])
   end
 
-  delegate :benefit_type, to: :applicant
+  def precedent_ioj_passport
+    if ioj_passport.include?('on_age_under18')
+      'on_age_under18'
+    elsif ioj_passport == ['on_offence']
```

Comment:
Apply is now catching if there is neither an Ioj reason, nor ioj passport, so we may not need this to be an explicit elsif, but it felt a bit safer. Although I wasn't sure what default else message to add, so it will blow up if Apply fails somehow